config/app.php

```diff
@@ -35,11 +35,11 @@
     |
     | This value determines the "environment" your application is currently
     | running in. This may determine how you prefer to configure various
-    | services the application utilizes. Set this in your ".env" file.
+    | services the application utilizes.
```

Comment:
Can we make this line  3 chars shorter than the previous one?

Reply:
Sorry, what do you mean? 🤔

Reply:
https://www.reddit.com/r/laravel/comments/3bifw5/every_doc_block_comment_is_3_characters_less_than/

Reply:
Never noticed that before. 😮 Interesting! I'll try and think of something...

Reply:
Maybe this?
```diff
    | This value determines the "environment" your application is currently
    | running in. This may determine how you prefer to configure various
-   | services the application utilizes. Set this in your ".env" file.
+   | services the application utilizes. This can be overridden using
+   | the global command line "--env" option when calling commands.
```

Reply:
LGTM